packages/back-end/src/controllers/datasources.ts

```diff
@@ -111,7 +111,7 @@ export async function deleteDataSource(
     );
   }
 
-  await deleteDatasourceById(datasource.id, org.id);
+  await deleteDatasourceById(datasource, org.id);
```

Comment:
This should either reverted or the name should be changed. I don't see why this change slipped through.